enhancements/declarative-index-config.md

```diff
@@ -421,15 +422,24 @@ $ cat etcd/etcd.json
 
 ### Story 2
 
-As an index author(with push permission to the namespace my index is hosted in), I can edit the packages in my index (remove bundles from a package, move a bundle to a different channel, etc), using only the json representation of the index/packages inside the index. 
+As an index author(with push permission to the registry/namespace my index is hosted in), I can edit the packages in my index (remove bundles from a package, move a bundle to a different channel, etc), using only the json representation of the index/packages inside the index. 
 
 ```bash
 $ cd community-operators 
 $ //edit etcd/etcd.json to add etcdoperator.v0.9.0 to a the channel `alpha`, and add an upgrade edge from etcdoperator-community.v0.6.1 to etcdoperator.v0.9.0 in channel `alpha`
 $ cd ..
-$ opm index update community-operators --tag=docker.io/my-namespace/community-operators:latest
+$ opm generate community-operators
+$ cat index.Dockerfile
+FROM quay.io/operator-framework/upstream-opm-builder
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+ADD community-operators /configs
+EXPOSE 50051
+ENTRYPOINT ["/bin/opm"]
+CMD ["serve","/configs"]
+$ docker build -f index.Dockerfile . --tag=docker.io/my-namespace/community-operators:latest
 $ rm -rf community-operators
-$ opm index inspect --image=docker.io/my-namespace/community-operators --output=json
+$ docker push docker.io/my-namespace/community-operators:latest
+$ opm unpack docker.io/my-namespace/community-operators 
```

Comment:
Is it necessary to show all of this for this story? I suspect some `tree` output from an existing directory containing an index would be sufficient.

Reply:
I suppose enumerating a flow for first time readers doesn't hurt too much